Workspace.py

```diff
@@ -638,11 +641,19 @@ def upload(
 
         # raises APIClientException
 
+        file_path = Utils.joinPath(self.getFullPath(), fileName)
+        if (
+            check_links.find_paths_links_file(file_path)
+            and self.apiClient.is_user_solo()
+        ):
+            raise APIClient.APIClientTierException(
+                "Uploading files with external links is not supported for tier Solo."
```

Comment:
The message is too esoteric.  Let's make it easier to understand.  I suggest:
"This document contains links to parts in another document.  Only single-document files are supported in your current lens account.  Please upgrade."

Reply:
Let's also avoid using the tier name 'solo' in the text string unless we're pulling it from lens.  Tier names might change in the future.

Reply:
Adapted it. The string does not contain "Please upgrade" because the dialog handles this with the following message: "Please upgrade your tier:" with a link to Ondsel Pricing.